### ISSUE TITLE
Add readBytes to read the raw image data in DicomImageReader

### ISSUE THREAD

**Issue by wayfarer3130** [dcm4che/dcm4che#204]
Sometimes it is desirable to allow directly extracting the raw byte array of the image in the dicom image reader, for example, if extracting JPEG images for display by a browser, or extracting the raw bytes to compare a second image decoding operation against the original one.

**Comment by er453r** [comment]
You can just read the `PixelData` attribute as `byte[]` to get the rawest image data possible

**Comment by er453r** [comment]
So you would like to read the raw pixel values, but **after** JPEG decompression?

(btw. I think I've found a solution to one your previous issues https://github.com/dcm4che/dcm4che/issues/51)

**Comment by wayfarer3130** [comment]
No, raw bytes as it is on disk, but as a single piece, regardless of
whether it came as a sub-section that is uncompressed, multi-segment image,
single-segment image in wrong location, or regular single-segment image.

**Comment by wayfarer3130** [comment]
The previous issue is resolved in the current trunk and in the
generic-config branches.
